### PR TITLE
PCORE-2058: creating multiple docker images for different builds

### PR DESCRIPTION
**Pull request by smit95tpatel**
Created two docker images for stage and prod builds

### CODE REVIEW

File: react-frontend/src/config/env.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export const env = {
```
Comment (sandeshdamkondwar) [inline react-frontend/src/config/env.js]: Why not using .env files?

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Set environment passed to container at runtime to global variable ENV.
+set -euo pipefail
+
+sed -i "s/<noscript id=\"env-insertion-point\"><\/noscript>/<script>var ENV=\"${APP_ENV}\"<\/script>/g" /app/react/build/index.html
```
Comment (sandeshdamkondwar) [inline docker-entrypoint.sh]: Use the .env files instead of this. Setup is already there to read from env files.

Comment (smit95tpatel) [reply docker-entrypoint.sh]: used env files

File: react-frontend/src/config/env.js (hunk 1 above)
Comment (smit95tpatel) [reply react-frontend/src/config/env.js]: used .env files

**Pull request closed without merging**